### PR TITLE
Update UG for editing a resident

### PR DESCRIPTION
**Pull request by teekoksiang**
Update UG for editing a resident

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -87,22 +87,19 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing a resident : `edit`
 
-Edits an existing person in the address book.
+Edits an existing resident in the Hall-y.
```
Comment (jieffyy) [inline docs/UserGuide.md]: Should it be just 'in Hall-y', instead of 'in _the_ Hall-y'?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -87,22 +87,19 @@ Shows a list of all persons in the address book.
...
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `edit 1 p/91234567 e/johndoe@example.com b/11 r/4` Edits the phone number, email address, block number and room number of the 1st resident to be 91234567, johndoe@example.com, 11 and 4 respectively.
+*  `edit 2 n/Betsy Crower p/87652103` Edits the name of the 2nd person and phone number to be `Betsy Crower` and 87652103 respectively.
```
Comment (jieffyy) [inline docs/UserGuide.md]: Should it be resident instead of person?

**Review by jieffyy — COMMENTED** [review]
Nice 👍 . Almost LGTM. A few changes (the person/resident changes can be quite hard to see) for your consideration.

**Comment by jieffyy** [comment]
Oops I missed this out: could you also update the command summary section for the `edit` command too?

**Comment by teekoksiang** [comment]
Update command summary and change 'person' to 'resident' in the User Guide.
Need to confirm about the CLI syntax for block and room number (combine together or separate).

File: docs/UserGuide.md (hunk 1 above)
Comment (teekoksiang) [reply docs/UserGuide.md]: Fixed, thanks.

**Comment by teekoksiang** [comment]
Update the CLI syntax for room number. r/ROOM_NUMBER
E.g. r/B402

**Review by jieffyy — APPROVED** [review]
LGTM 👍 Merging!

**Pull request merged**